examples.yaml

```
@@ -93,7 +93,7 @@
       id: extrema
       content: |
         import yt
-        ds = yt.load("Enzo_64/DD0043/data0043")
+        ds = yt.load_sample("Enzo_64", "DD0043/data0043")
```

With the current version of load_sample, this is how you need to write this :)
```suggestion
        ds = yt.load_sample("Enzo_64/DD0043/data0043")
```

---

still relevant

---

Right -- this *used* to work, and no longer does.